src/renderer/components/top-nav/top-nav.sass

```diff
@@ -85,7 +85,22 @@
     display: flex
     align-items: center
     padding: 0px 25px 0px 10px
+    cursor: pointer
+    @include top-nav-is-colored
+      color: var(--text-with-main-color)
+
+      &:hover
+        background-color: var(--primary-color-hover)
 
+    &:hover
+      background-color: var(--side-nav-hover-color)
+      transition: background 0.2s ease-in
```

Comment:
Hi, I'm not a fan of the background color changing. It could just be a cursor: pointer on hover and that is fine with me.

Reply:
This makes it more consistent with the other nav icons https://github.com/FreeTubeApp/FreeTube/blob/development/src/renderer/components/top-nav/top-nav.sass#L41
to
https://github.com/FreeTubeApp/FreeTube/blob/development/src/renderer/components/top-nav/top-nav.sass#L56

Reply:
I understand, but normally the logo is an alternate way of clicking "home". The logo therefore is not really treated like a button, but as a secondary shortcut.